### PR TITLE
This is my Second Commit - Dhana

### PR DESCRIPTION
**Pull request by hidhana**

### CODE REVIEW

**Comment by cjalkam** [comment]
Approved

File: README.md [hunk 1]
```diff
@@ -58,3 +58,7 @@ After you have completed this course, you are probably wondering where to go fro
 - [GitHub Learning Lab](https://lab.github.com/): self-paced courses with instant bot-assisted feedback.
 - [Microsoft Learn for GitHub](https://docs.microsoft.com/en-us/learn/github/)
 - [Join the open source community](https://github.com/open-source)
+
+### Message
+
+Hi. This is Dhana!
```
Comment (JesusNavarrete215) [inline README.md]: Good to go !!!

**Comment by cjalkam** [comment]
Approved

**Review by JesusNavarrete215 — APPROVED** [review]

**Comment by cjalkam** [comment]
Approved

**Comment by cjalkam** [comment]
Approved

**Comment by cjalkam** [comment]
Approved

**Comment by cjalkam** [comment]
Approved

File: README.md (hunk 1 above)
Comment (cjalkam) [reply README.md]: Looks good

**Review by cjalkam — APPROVED** [review]

**Pull request merged**